tests/components/utility_meter/test_sensor.py

```diff
@@ -638,82 +638,89 @@ async def test_restore_state(
     # Home assistant is not runnit yet
     hass.set_state(CoreState.not_running)
 
-    last_reset = "2020-12-21T00:00:00.013073+00:00"
+    last_reset_1 = "2020-12-21T00:00:00.013073+00:00"
+    last_reset_2 = "2020-12-22T00:00:00.013073+00:00"
```

Comment:
Use different timestamps so we know if the timestamp is restored from the `UtilitySensorExtraStoredData`